jsk_robot_common/jsk_robot_startup/lifelog/tweet_client_uptime.l

```diff
@@ -18,6 +18,13 @@
   (unix::sleep 3)
   (ros::ros-info "Wait for /active_user/elapsed_time parameter ..."))
 
+(cond
+ ((ros::has-param "/active_user/tweet_random_range")
+  (setq *tweet-random-range* (ros::get-param "/active_user/tweet_random_range")))
+ (t
+  (setq *tweet-random-range* 600.0)
```

Comment:
I think this section sets random range without explicitly setting rosparm `/active_user/tweet_random_range`

Reply:
I see, you're right.